Resources/Prototypes/_NF/Entities/Objects/Vehicles/vehicles.yml

```diff
@@ -58,19 +58,24 @@
   name: skeleton motorcycle
   description: Bad to the Bone.
   components:
-    - type: Storage
-      grid:
-      - 0,0,1,3
-      - 3,0,6,3
-      - 8,0,9,3
-      maxItemSize: Huge
-    - type: ContainerContainer
-      containers:
-        storagebase: !type:Container
-    - type: UserInterface
-      interfaces:
-      - key: enum.StorageUiKey.Key
-        type: StorageBoundUserInterface
+  - type: Storage
+    grid:
+    - 0,0,1,3
+    - 3,0,6,3
+    - 8,0,9,3
+    maxItemSize: Huge
+    blacklist:
+      components:
+      - Felinid
+      - Vulpakin
```

Comment:
There is no comp named Vulpakin,
Let me find how to fix this..

Reply:
I can probably use `MindContainer` instead of listing species. It's more general so should futureproof this blacklist too, should any changes to sentient species sizes come.

Reply:
Made the change, felinids/goblins/vulps/brains are banned from dwelling in hoverbike storage. MMI and pAIs are fine.